src/components/navigation/navigation-item/NavigationItem.tsx

```diff
@@ -161,37 +137,6 @@ export class NavigationItem extends Component<NavigationItemProps, NavigationIte
     return <img {...imageProps} />;
   };
 
-  private _renderShowMoreLessButton = (title: string) => {
-    if (!this.props.data.previewImage) {
-      return null;
-    }
-    const {expandText} = this.state;
-    return (
-      <A11yWrapper onClick={this._handleExpandChange}>
-        <div
-          tabIndex={0}
-          className={styles.showMoreButton}
-          ref={node => {
-            if (expandText) {
-              this._showLessButtonRef = node;
-            } else {
-              this._showMoreButtonRef = node;
-            }
-          }}
-          aria-label={`${expandText ? this.props.readLessLabel : this.props.readMoreLabel} ${title}`}>
-          {expandText ? this.props.readLessTranslate : this.props.readMoreTranslate}
```

Comment:
readLessTranslate,  readLessLabel, readMoreTranslate and readMoreLabel moved to `ExpandableText` component, can we remove translates from Navigation plugin

Reply:
Oh, we already have translations ! I'll add them to playkit-ui then

Reply:
When was show_more_of used vs show_more ?

Reply:
according to code:
`show_more_of` uses in aria-label
`show_more` displays as button text